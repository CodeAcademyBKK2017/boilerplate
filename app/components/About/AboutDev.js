import React from 'react';
import {
  Text
} from 'react-native';
  
export default class MyNotificationsScreen extends React.Component {
    static navigationOptions = {
      tabBarLabel: 'DEV'
    };
    render () {
      return (
        <Text>DEV</Text>
        // <Button
        //   onPress={() => this.props.navigation.goBack()}
        //   title='Go back home'
        // />
      );
    }
}
