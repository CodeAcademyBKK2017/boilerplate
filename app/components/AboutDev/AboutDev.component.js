import React, {Component} from 'react';
import styles from './AboutDev.style';
import {Text, View} from 'react-native';

export default class AboutDevScreen extends Component {
    static navigationOptions = {
      title: 'About Developer'
    };
      
    render () {
      return (
        
        <View style={styles.container}>
          <Text>About Developer</Text>
        </View>
      );
    }
  
}