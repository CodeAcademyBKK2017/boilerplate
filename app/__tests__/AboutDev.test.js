import 'react-native';
import AboutDev from '../components/About/AboutDev.component';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
// import {shallow} from 'enzyme';

describe('AboutDev', () => {
  it('renders correctly AboutDev', () => {
    const tree = renderer.create(
      <AboutDev />
    );
    expect(tree).toMatchSnapshot();
  });
});