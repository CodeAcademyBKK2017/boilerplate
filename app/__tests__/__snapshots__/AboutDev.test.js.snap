// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutDev renders correctly AboutDev 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    About the Creators
  </Text>
</View>
`;
